bin/update-yarn.sh

```diff
@@ -21,7 +21,9 @@ PLUGINS=$(awk '{ if ($1 == "spec:") print $2 }' .yarnrc.yml)
 echo "$PLUGINS" | xargs -n1 -t corepack yarn plugin remove
```

Comment:
maybe add a comment to the top of this script. I'm not 100% sure what is the purpose of it. Is it run from github actions? Or just a manual helper script to run on dev machine

Reply:
Something like that?
This script is meant to be run on a dev machine to update the version on Yarn used by the monorepo. Its goal is to make sure that every mention of Yarn version is updated, and it re-installs the plugins to make sure those are up-to-date as well.

Reply:
Ok, great :) would be nice to add that as a comment at the top of the script